src/logic/config/store/reducer/index.ts

```diff
@@ -0,0 +1,31 @@
+import { Action, handleActions } from 'redux-actions'
+
+import { AppReduxState } from 'src/store'
+import { CONFIG_ACTIONS } from 'src/logic/config/store/actions'
+import { makeNetworkConfigFromSpecificConfiguration, NetworkState } from 'src/logic/config/model/networkConfig'
+import { getConfig, getNetworkId, getNetworkName, NetworkSpecificConfiguration, setNetworkId } from 'src/config'
+import { ETHEREUM_NETWORK } from 'src/config/networks/network.d'
+
+export const NETWORK_CONFIG_REDUCER_ID = 'networkConfig'
+
+type Payload = NetworkState | NetworkSpecificConfiguration | string
+
+export default handleActions<AppReduxState['networkConfig'], Payload>(
+  {
+    [CONFIG_ACTIONS.CONFIG_STORE]: (state, action: Action<NetworkSpecificConfiguration>) => {
+      const prevState = { ...state }
+      const newParams = makeNetworkConfigFromSpecificConfiguration(action.payload)
```

Comment:
Would be also good to move this out to the action

Reply:
All this should change once we transition to the config-service, that's why I did more dirty things in this reducer, only for my convenience.